### ISSUE TITLE
Action `redirect_to` should use Turbo by default

### ISSUE THREAD

**Issue by marcoroth** [marcoroth/turbo_power#13]
Currently you need to pass `turbo="true"` to the `<turbo-stream>` element in order that is uses `Turbo.visit()`:

```html
<turbo-stream turbo="true" url="http://localhost:3000/" action="redirect_to"><template></template></turbo-stream>
```

It should use `Turbo.visit()` by default unless `turbo="false"` gets explicitly passed to the stream element.

The action already accounts for when Turbo/Turbolinks is not installed in the application and will fallback to `window.location.href = url`

https://github.com/marcoroth/turbo_power/blob/623a6f5b11608073ac36880bd1f0cc4745a52e56/src/actions/browser.ts#L17

**Comment by pySilver** [comment]
I'm wondering what is the correct way to pass value for `action` (_advance/replace_) since it reserved keyword.

**Comment by marcoroth** [comment]
You are right, @pySilver! @leon-vogt also pointed this out to me some time ago, but I haven't been able to change it yet.

We should rename this attribute to `action_name`. 

Feel free to open a PR if you like! 🙌🏼

**Comment by marcoroth** [comment]
I opened a separate issue for this: #15

**Comment by marcoroth** [comment]
> I'm wondering what is the correct way to pass value for `action` (_advance/replace_) since it reserved keyword.

@pySilver this was resolved via #22 is was released as part of [0.1.6](https://github.com/marcoroth/turbo_power/releases/tag/v0.1.6)

**Comment by pySilver** [comment]
@marcoroth thank you for your work!
